pkg/bucketeer/sdk.go

```diff
@@ -269,18 +269,19 @@ func (s *sdk) callGetEvaluationAPI(
 		gserr = err // set HTTP status error
 		code, ok := api.GetStatusCode(gserr)
 		if !ok {
-			s.eventProcessor.PushInternalErrorCountMetricsEvent(ctx)
+			s.eventProcessor.PushInternalErrorMetricsEvent(ctx, model.GetEvaluation)
 			return nil, fmt.Errorf("failed to get evaluation: %w", err)
 		}
 		if code == http.StatusGatewayTimeout {
```

Comment:
Can you also add a case for network errors? Such as connection refused error.
https://github.com/bucketeer-io/bucketeer/blob/main/proto/event/client/event.proto#L172-L175

Reply:
For connection refused, you can do something like this.
https://go.dev/play/p/CvgfnU8QdQ5

Also, maybe there is a way to handle the unknown host error.